### DISCUSSION TITLE
Explaination of the errors and suggested fixes?

### DISCUSSION THREAD

**Discussion by briandipalma** [ansible/ansible-language-server#108] [Ideas]
All my ansible files seem to be terrible even though they work, now I'm happy to improve them but I don't have the knowledge to know what fixes I should apply to all these errors. Googling doesn't seem to be too helpful either. I mean for example:
![image](https://user-images.githubusercontent.com/1597820/140501172-7b91d2bc-40fb-4139-8850-cf3b0ab071d6.png)
Why? I have no vars to use in this case, I'm baffled as to what I'm doing wrong. Maybe some documentation explaining these errors would be helpful, similar to ESLint's website.

**Comment by webknjaz** [comment]
This one is probably coming from `ansible-lint`. Could you please verify this by running the lint directly and open a feature request there, if it is. I'm pretty sure that ALS re-exposes lint's messages.

**Comment by briandipalma** [comment]
You're right, I didn't have ansible-lint installed which made it even worse as I was seeing weird syntax check errors. Once I installed it everything improved a lot. Sorry about that.
